java/src/main/java/com/ncc/aif/ThreadedValidationEngine.java

```diff
@@ -207,7 +209,7 @@ private boolean exceedsMaximumNumberViolations(int violations) {
 //        if (monitor != null) {
 //            monitor.beginTask("Validating " + rootShapes.size() + " shapes", rootShapes.size());
 //        }
-
+            logger.debug("Validating " + rootShapes.size() + " shapes.");
```

Comment:
For the debug statements, please use parameterized messages: http://www.slf4j.org/faq.html#logging_performance

Reply:
Ah, yes-- thanks.  Fixed.